src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/PreparedStatementTest.java

```diff
@@ -43,8 +45,9 @@
 @RunWith(JUnitPlatform.class)
 public class PreparedStatementTest extends AbstractTest {
 
-    final String tableName = RandomUtil.getIdentifier("#update1");
-    final String tableName2 = RandomUtil.getIdentifier("#update2");
+    final String tableName = RandomUtil.getIdentifier("test1");
+    final String tableName2 = RandomUtil.getIdentifier("test2");
+    final String tableName3 = RandomUtil.getIdentifier("test3");
```

Comment:
I know this had stupid names before too but can we name them to something more meaningful so we know what the test the table is used for if find them left in the database.